flow/e2e/test_utils.go

```diff
@@ -671,7 +671,10 @@ func (env WorkflowRun) Query(ctx context.Context, queryType string, args ...any)
 
 func (env WorkflowRun) GetFlowStatus(t *testing.T) protos.FlowStatus {
 	t.Helper()
-	flowStatus, err := internal.GetWorkflowStatus(t.Context(), env.c, env.GetID())
```

Comment:
why does this change? bypassing cache?

Reply:
e2e tests don't end up storing in catalog at all, so we get UNKNOWN status
bypassing that logic

Reply:
I'm not sure, but I think we should not bypass cache during tests

@heavycrystal changed this I think